spec/models/food_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+RSpec.describe Food, type: :model do
+  it 'belongs to a user' do
+    food = Food.new
+    expect(food).to respond_to(:user)
+  end
+
+  it 'has a name' do
+    food = Food.create(name: 'Pizza')
+    expect(food.name).to eq('Pizza')
+  end
+end
```

Comment:
- [ ] Kindly implement more or as many tests as you can for your application.

  - food_spec
  - login_in_spec
  - recipe_foods_spec
  - recipes_spec
  
  Please implement tests for those files as well as test the functionalities of your application.

Reply:
Tests created for: user sign in, foods, recipe, recipe foods